docs/docs/event-histories/README.md

```diff
@@ -0,0 +1,88 @@
+# Event History
+
+Monitor all workflow events and their stace traces in one centralized view under the [**Event History**](https://pipedream.com/event-history) section in the dashboard.
+
+Within the **Event History**, you'll be able to filter your events by workflow, execution status, within a specific time range.
+
+This includes events in progress for time time visibility into the health of your workflows.
+
+[[toc]]
+
+## Filtering Events
+
+The filters at the top of the screen allow you to search all events processed by your workflows:
+
+![Filtering all events in a workspace by it's status, when it was executed or by workflow ID](https://res.cloudinary.com/pipedreamin/image/upload/v1682696567/docs/docs/event%20histories/CleanShot_2023-04-28_at_11.42.19_d8bfer.png)
+
+You can filter by the event's **Status**, **time of initiation** or by the **Workflow ID**.
+
+::: tip 
+
+The filters are scoped to the current [workspace](/workspaces/).
+
+If you're not seeing the events or workflow you're expecting, try [switching workspaces](workspaces/#switching-between-workspaces).
+
+:::
+
+### All failed workflow executions
+
+You can view all failed workflow executions by applying the **Error** status filter.
+
+This will only display the failed workflow executions in the selected time period.
+
+This view in particular is helpful for identifying trends of errors, or workflows with persistent problems.
+
+### Within a time range
+
+Filtering by time range will only include workflow events _started_ within the define range.
+
+::: tip Long running workflows
+
+The time range filter depends on when the execution event first started. If a long running workflow starts within the specified time range but it's execution continues _past_ the filtered timeframe, it will still be included in the results.
```

Comment:
```suggestion
The time range filter depends on when the execution event first started. If a long-running workflow starts within the specified time range but its execution continues _past_ the filtered timeframe, it will still be included in the results.
```

Reply:
Changed "it's execution" to "its execution" (possessive pronoun)